hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -35,12 +43,24 @@ Check that the output `0x3020000a` is shown. If it is not, then the OTP bit has
 
 If you wish, you can remove the `program_usb_boot_mode` line from config.txt, so that if you put the SD card into another Raspberry Pi, it won't program USB host boot mode. Make sure there is no blank line at the end of config.txt.
 
-## Prepare the USB mass storage device
+You can now boot from an USB mass storage device in the same way as booting from SD, see the following paragraph for Raspberry Pi 3B+, CM3+.
+
+## Raspberry Pi 3B+, CM3+
+
+The Raspberry Pi 3B+ and CM3+ support USB mass storage boot out of the box. The settings specific to the previous versions of Raspberry Pi do not have to be executed.
+
+This is the same [procedure](../../../installation/installing-images) as for SD cards.
```

Comment:
"This is the same procedure..." feels a bit out of context IMHO? Perhaps a slight tweak of previous text like "Copy a Raspbian or Raspbian Lite installation image onto your USB device using the same procedure..." would be better?

Reply:
You've just reviews a PR that has already merged, just to make you aware.

Reply:
I realise that _now_, but I think that's because I was making comments at the same time that James was merging it :wink:

Reply:
> I realise that _now_, but I think that's because I was making comments at the same time that James was merging it 😉

Indeed